### PR TITLE
MH-12829 Switch and rename event details tabs

### PR DESCRIPTION
**Pull request by KatrinIhler**
This PR switches the metadata and general tabs in the event details modal so that the metadata tab is in front. This accomplishes two things: 1. It's consistent with the series details modal, and 2. it makes more sense in my opinion to view the metadata first.

Additionally the general tab is renamed to publications tab since it contains publications and nothing else. For consistency a number of files and identifiers are also renamed. Because of naming conflicts (see discussion on list here) I thought it best to rename the resources for the assets tab to contain 'Asset' in their name, thus avoiding conflicts concerning 'eventPublicationsResource'. (I hope I didn't miss anything during my renaming spree, if I did, just comment and I'll fix it.)

See the screenshot for more details: 

screenshot from 2018-05-07 16-46-19

_This work is sponsored by ETH._

### CODE REVIEW

File: etc/security/mh_default_org.xml [hunk 1]
```diff
@@ -53,7 +53,7 @@
     <sec:intercept-url pattern="/admin-ng/event/*/attachments.json" method="GET" access="ROLE_ADMIN, ROLE_UI_EVENTS_DETAILS_ATTACHMENTS_VIEW" />
     <sec:intercept-url pattern="/admin-ng/event/*/comments" method="GET" access="ROLE_ADMIN, ROLE_UI_EVENTS_DETAILS_COMMENTS_VIEW"/>
     <sec:intercept-url pattern="/admin-ng/event/*/comment/*" method="GET" access="ROLE_ADMIN, ROLE_UI_EVENTS_DETAILS_COMMENTS_VIEW"/>
-    <sec:intercept-url pattern="/admin-ng/event/*/general.json" method="GET" access="ROLE_ADMIN, ROLE_UI_EVENTS_DETAILS_GENERAL_VIEW" />
+    <sec:intercept-url pattern="/admin-ng/event/*/publications.json" method="GET" access="ROLE_ADMIN, ROLE_UI_EVENTS_PUBLICATIONS_GENERAL_VIEW" />
```
Comment (staubesv) [inline etc/security/mh_default_org.xml]: Remove "GENERAL"

**Review by staubesv — CHANGES_REQUESTED** [review]
Hi @KatrinIhler 

Just found one little typo (wrong role name in mh_default_org.xml). Except for this, lgtm.

File: etc/security/mh_default_org.xml (hunk 1 above)
Comment (KatrinIhler) [reply etc/security/mh_default_org.xml]: Thanks, fixed!

**Comment by KatrinIhler** [comment]
@staubesv I had also missed to check in the changes I made to etc/listproviders/publication.channels.properties, fixed that, too.

**Comment by staubesv** [comment]
Hi @KatrinIhler I'm currently testing this and found a first issue: If you click on the "Yes" in the Events table for a published event, the Event Detail modal opens the tab metadata. As the user clicks on a cell in the column "Published", he might expect the tab "Publications" to be opened.

**Comment by KatrinIhler** [comment]
@staubesv This was broken already, but wasn't visible until now - fixed!

**Review by staubesv — APPROVED** [review]

**Pull request merged**